src/core/waf.cc

```diff
@@ -46,6 +46,25 @@
 #include <unistd.h>
 #include <dirent.h>
 #include <errno.h>
+
+namespace {
+/// Mark the context with the applied tx. This can be used to avoid performing the
```

Comment:
We're planning on "re-doxygen-ify'ing" our code comments soon, but just to be consistent with the other function comment blocks -may something like: (where the ascii art is 80 characters wide)
```cpp
//: ----------------------------------------------------------------------------
//: \details: my cool function is cool
//: ----------------------------------------------------------------------------
```